1.9/overview/concepts.md

```diff
@@ -348,7 +348,7 @@ A Mesos resource offer provides a set of unallocated resources (e.g. cpu, disk,
 
 ### <a name="mesos-containerizer"></a>Containerizer
```

Comment:
We may also want to fix this subtitle as well. Could we follow a commit to fix it?